frontend/src/layouts/GLogin.vue

```diff
@@ -184,7 +184,10 @@ export default {
     if (this.error) {
       const err = this.error
       this.error = null
-      this.setError(err)
+      this.setError({
```

Comment:
while this fixes the issue, shouldn't we enhance the `setAlertWithType` in `appStore` to also support the `message` property as alternative to `text`. This ensures that the function works with error objects and prevents possible wrong usage in the future

Reply:
passing an error object is now supported